src/scripts/main.js

```diff
@@ -1,3 +1,144 @@
 'use strict';
 
-// write code here
+const tableHeaders = document.querySelectorAll('thead th');
+const tableBody = document.querySelector('table');
```

Comment:
The `tableBody` variable should select the `tbody` element instead of the `table`. Change the selector to `document.querySelector('tbody')` to ensure that only the body rows are targeted for sorting.